src/Crypto.Websocket.Extensions.Core/Orders/CryptoOrders.cs

```diff
@@ -10,25 +10,26 @@
 using Crypto.Websocket.Extensions.Core.Utils;
 using Crypto.Websocket.Extensions.Core.Validations;
 
+
 namespace Crypto.Websocket.Extensions.Core.Orders
 {
     /// <summary>
-    /// Orders manager
+    ///     Orders manager
```

Comment:
Please avoid changing the existing code as much as possible (also below with order switching).